src/app/book/store/actions/book.actions.ts

```diff
@@ -0,0 +1,29 @@
+import { createAction, props } from '@ngrx/store';
+import { BookModel } from '../../../core/models/book';
+
+export const GetBooks = createAction (
```

Comment:
you need to get it from backend, so better rename FetchBooks

Reply:
fixed